src/styles/animations.ts

```diff
@@ -2,18 +2,18 @@ import { keyframes } from 'styled-components';
 
 export const slideUp = keyframes`
   0% {
-    bottom: -300px;
+    bottom: -100%;
   }
   100% {
-    bottom: 0;
+    bottom: 0%;
   }
 `;
 
 export const slideDown = keyframes`
   0% {
-    bottom: 0;
+    bottom: 0%;
   }
   100% {
-    bottom:-300px
+    bottom:-100%;
```

Comment:
```suggestion
    bottom: -100%;
```

키프레임은 프리티어 안 먹는게 아쉽네요 🥲

Reply:
아 맞다 ㅎ....깜박